### ISSUE TITLE
«NameError: uninitialized constant SiteSettings::DefaultsProvider::DistributedCache» on `bundle exec rake db:migrate` after upgrading to Discourse v1.9.0.beta11

### ISSUE THREAD

**Issue by dmitrii-fediuk** [discourse-pro/df-core#1]
![03](https://user-images.githubusercontent.com/83204/34713406-5e7d4e1a-f550-11e7-90d5-ef58d5320ebe.png)

```
rake aborted!
NameError: uninitialized constant SiteSettings::DefaultsProvider::DistributedCache
lib/site_settings/defaults_provider.rb:12:in `<class:DefaultsProvider>'
lib/site_settings/defaults_provider.rb:4:in `<main>'
lib/site_setting_extension.rb:3:in `<main>'
plugins/df-core/plugin.rb:96:in `activate!'
lib/plugin/instance.rb:397:in `instance_eval'
lib/plugin/instance.rb:397:in `activate!'
lib/discourse.rb:146:in `block in activate_plugins!'
lib/discourse.rb:143:in `each'
lib/discourse.rb:143:in `activate_plugins!'
config/application.rb:178:in `<class:Application>'
config/application.rb:26:in `<module:Discourse>'
config/application.rb:25:in `<top (required)>'
rakefile:5:in `require'
rakefile:5:in `<top (required)>'
/home/dfediuk/.rbenv/versions/2.4.2/bin/bundle:23:in `load'
/home/dfediuk/.rbenv/versions/2.4.2/bin/bundle:23:in `<main>'
```

**Comment by dmitrii-fediuk** [comment]
[Why does the `SiteSettings` class use the `DistributedCache` without a `require 'distributed_cache';` statement?](https://meta.discourse.org/t/77580)

**Comment by dmitrii-fediuk** [comment]
Fixed in [1.2.5](https://github.com/discourse-pro/df-core/releases/tag/1.2.5).
